litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-LIMIT-RESOURCE/test.yml

```diff
@@ -27,13 +27,61 @@
           shell: cat /etc/secret-volume/password
           register: password
 
-        ## Checking if openebs is already installed
+        ## Checking if openebs is already installed in the cluster
         - name: Check if openebs is already installed
```

Comment:
we are checking only maya-apiserver here

Reply:
yes we are only checking for maya-apiserver pod and if it is present then in next step we are fetching the ns and then deleting all the components